applications/crossbar/src/api_util.erl

```diff
@@ -393,22 +396,26 @@ default_filename() ->
                            halt_return().
 decode_base64(Context, CT, Req0) ->
     decode_base64(Context, CT, Req0, []).
+
 decode_base64(Context, CT, Req0, Body) ->
     case cowboy_req:body(Req0) of
         {'error', 'badlength'} ->
             lager:debug("the request body was most likely too big"),
             ?MODULE:halt(Req0
-                         ,cb_context:add_validation_error(<<"file">>, <<"maxLength">>
+                         ,cb_context:add_validation_error(<<"file">>
+                                                          ,<<"maxLength">>
                                                           ,<<"File was larger than allowed">>
                                                           ,Context
-                                                          ));
+                                                         )
```

Comment:
Should this not also be a 413?
cb_context:set_resp_error_code(Context, 413)

Maybe check all the modules to make sure it is consistent?